autotst/data/update.py

```diff
@@ -67,10 +67,8 @@ def get_unknown_species(reactions, known_species):
 
     for i, reaction in enumerate(reactions):
         rmg_reaction = reaction.rmg_reaction
-        r1, r2 = rmg_reaction.reactants
-        p1, p2 = rmg_reaction.products
 
-        relavent_species = [r1, r2, p1, p2]
+        relavent_species = rmg_reaction.reactants + rmg_reaction.products
         relavent_labels = {}
 
         for rel_species in relavent_species:
```

Comment:
If these are labeled species, then I think we should be using in initial map with the atom labels when checking isomorphism.

Reply:
@rwest comment: there's an `generate_initial_map` option to most isomorphism methods that generates a mapping by matching labels, so you don't need to make your own map. Might be worth trying

Reply:
Okay, after looking through this a bit more, this chunk of the code is simply designed to find a list of species that need to be added to a dictionary containing both SMILEs strings as keys and RMG Molecule objects as values. This dictionary is then used to generate SMILEs string labels used for creating reaction strings (`r1 + r2 = p1 + p2`) when adding comments to the saved TS database. But good catch on if we need the labels or not.